pkg/apis/core/helper/helpers.go

```diff
@@ -167,6 +167,7 @@ var overcommitBlacklist = sets.NewString(string(core.ResourceNvidiaGPU))
 func IsOvercommitAllowed(name core.ResourceName) bool {
 	return IsDefaultNamespaceResource(name) &&
 		!IsHugePageResourceName(name) &&
+		!IsExtendedResourceName(name) &&
```

Comment:
This predicate is technically correct but redundant since extended resources are by definition disjoint from default namespace resources. Probably we only need one of the two, but it would be helpful to add a comment saying why this was correct.

Reply:
Agree, comment makes more sense over redundant line of code.

Reply:
Good catch! Removed.